vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oauth2/OAuth2Client.kt

```diff
@@ -38,19 +39,22 @@ class OAuth2Client(
      * and then [TokenRequestParameters.codeVerifier], see [RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636).
      */
     private val stateToCodeStore: MapStore<String, String> = DefaultMapStore(),
+    @Deprecated("Use signPushedAuthorizationRequest instead")
```

Comment:
add `replaceWith="signPushedAuthorizationRequest"` to the annotation

Reply:
But it's not really an exact replacement, so I've not included that